rollup.config.js

```diff
@@ -27,6 +27,8 @@ function getEsbuild(target) {
     minify: false,
     target,
     tsconfig: path.resolve('./tsconfig.json'),
+    allowSyntheticDefaultImports: true,
+    module: 'esnext',
```

Comment:
what problem do these solve?

Reply:
This one, I missed to reset, was in correspondence to the tsconfig change I made earlier